src/datasources/system/SystemDataSource.ts

```diff
@@ -40,6 +41,13 @@ export class SystemDataSource extends DataSourceApi<SystemQuery> {
     return NetworkUtils.getIpAddressFromInterfaces(ip4Interface) || NetworkUtils.getIpAddressFromInterfaces(ip6Interface);
   }
 
+  async metricFindQuery(query: VariableQuery, options?: any): Promise<MetricFindValue[]> {
+    const response = await getBackendSrv().post<{ data: VariableQuery[] }>(this.baseUrl + '/query-systems', { projection: "new(id, alias)" });
+    const values = response.data.map(frame => ({ text: frame.alias, value: frame.id }));
+
+    return values;
```

Comment:
The return type annotation states `Promise<MetricFindValue[]>`, but I don't think that is the type of this object?